compiler/luci/lang/include/luci/IR/CircleShapeSignature.h

```diff
@@ -46,6 +46,8 @@ class ShapeSignature
   std::vector<int32_t> _shape_signature{};
 };
 
+bool operator==(const ShapeSignature &lhs, const ShapeSignature &rhs);
```

Comment:
(optional) This one looks good as well:-)
```suggestion
...
public:
bool operator==(const ShapeSignature &rhs)
{
  return _shape_signature == rhs._shape_signature;
}

//bool operator==(const ShapeSignature &lhs, const ShapeSignature &rhs)
//{
//  if (lhs.rank() != rhs.rank())
//    return false;
//
//  for (uint32_t i = 0; i < lhs.rank(); ++i)
//    if (lhs.dim(i) != rhs.dim(i))
//      return false;
//
//  return true;
//}
```